app/ports/templates/ports/about.html

```diff
@@ -0,0 +1,73 @@
+                    <li><a href="https://github.com/macports/macports-webapp/tree/master/docs">Documentation</a></li>
+                    <li><a href="https://github.com/macports/macports-webapp/tree/master/docs/api_v1">API</a></li>
+                    <li><a href="https://docs.google.com/document/d/1TqNHm3NrtLDelWqY7P4pZyTMZDJrlsPymFZklzV2t24/edit">Feature Requests</a></li>
+                    <li><a href="https://github.com/macports/macports-webapp/issues">Issues</a></li>
+                    <li><a href="https://www.macports.org/install.php">Install MacPorts</a></li>
+                </ul>
+        </div>
+        <div class="col-lg-9 col-lg-8">
+            The MacPorts web application:
+            <ol>
+                <li>Displays general information about all the ports.</li>
+                <li>Fetches and displays build history of the ports.</li>
+                <li>Collects and displays installation statistics (system configurations and installed ports) from users
+                    who opt-in by installing the port mpstats.
+                </li>
+                <li>Makes available all the collected data through API.</li>
+            </ol>
+            <p>The application is based on the <a href="https://www.djangoproject.com">Django Framework</a> and runs
+                inside a <a href="https://www.docker.com">Docker</a> container with <a href="https://www.nginx.com">nginx</a>
+                and <a href="https://uwsgi-docs.readthedocs.io">uWSGI</a> serving the content. It has been written as a part of the <a href="https://summerofcode.withgoogle.com">
+                Google Summer of Code 2019</a> mentored by the <a href="https://www.macports.org">MacPorts</a> organisation.</p>
+        </div>
+    </div>
+    <br>
+    <h4>Credits</h4>
+    <hr>
+    <p>The MacPorts Webapp was written by <a href="https://github.com/arjunsalyan">Arjun Salyan</a> when MacPorts
+        participated in the Google Summer of Code 2019. The project was mentored by
+        <a href="https://github.com/mojca">Mojca Miklavec</a> and <a href="https://github.com/umeshksingla">Umesh Singla</a>.
+        The server is administered by <a href="https://github.com/neverpanic">Clemens Lang</a>.
+    </p>
+
+    <h5>Old Statistics Website</h5>
+    <p>
+	The old MacPorts statistics web application and the initial client were written by <a href="https://trac.macports.org/wiki/derek">Derek Ingrouville</a>
+	when MacPorts participated in <a href="https://trac.macports.org/wiki/SummerOfCode2011">Google Summer of Code 2011</a>. Some documentation for the
+	source written in GSoC can be found in <a href="https://trac.macports.org/wiki/MacPortsStatisticsGSoC2011">the Wiki</a>. The source code for the
+	server is available in MacPorts' Subversion repository at <a href="https://trac.macports.org/browser/branches/gsoc11-statistics">/branches/gsoc11-statistics</a>.
+    </p>
+    <br>
+    <h4>Dependencies</h4>
```

Comment:
Maybe also technologies (Django etc.) could be mentioned at this place (moved from the start). Not necessarily under this section, but somewhere around here.

Reply:
Done, thanks.